src/core/packages/chrome/browser-internal/src/ui/project/sidenav_v2/navigation/to_navigation_items.tsx

```diff
@@ -189,14 +189,14 @@ export const toNavigationItems = (
             items: navNode.children.map((child) => {
               warnUnsupportedNavNodeOptions(child);
               maybeMarkActive(child, 2);
-              return {
+              const item: SecondaryMenuItem = {
                 id: child.id,
+                isExternal: child.isExternalLink,
                 label: warnIfMissing(child, 'title', 'Missing Title 😭'),
                 href: warnIfMissing(child, 'href', 'Missing Href 😭'),
-                external: child.isExternalLink,
-                iconType: child.icon,
```

Comment:
@Dosant I'm wondering about this here. Do we already use the "Add data" pattern? Do we need a user-defined left-side icon?